programs/voter-stake-registry/src/state/lockup.rs

```diff
@@ -10,15 +10,15 @@ vote_weight_record!(crate::ID);
 
 /// Seconds in one day.
 #[cfg(feature = "localnet")]
-pub const SECS_PER_DAY: i64 = 10;
+pub const SECS_PER_DAY: u64 = 10;
```

Comment:
comment: haha why didnt we start with unsigned in first place, now looks obviously better

Reply:
Agree. I'd like it best if solana::Clock::unix_timestamp was `u64` and not `i64`...